docs/kit/kit-definition.md

```diff
@@ -0,0 +1,57 @@
+---
+original:
+  authors: Richard Barlow
+  url: https://bitbucket.org/richardbarlow/sr-kit-coord/wiki/Kit%20Definition
+---
+# Kit Definition
+
+This page serves as the canonical list of parts constituting a Student Robotics kit.
+
+## Hardware
+
+Part                     | Inventory Asset Type  | Qty |
```

Comment:
We should link to the docs page for this, save duplicating information

https://studentrobotics.org/docs/kit/

Reply:
I'm not sure I agree -- this page is arguably canonical and more technical. In practise it might not be worth the duplication though.